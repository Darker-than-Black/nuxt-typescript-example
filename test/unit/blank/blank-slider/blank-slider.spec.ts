import VueTestInstance from '~/test/factory/vue-instance'
import BlankSlider from '~/components/blank/items/blank-slider/index.vue'
import DECLARATION_BLANK_ITEMS from '~/assets/js/glossary/balnk/declaration-blank-items'
import DeclarationMock from '~/test/api/declaration-info.json'

describe('blank-slider/index.vue', () => {
  let wrapper: any

  beforeEach(() => {
    wrapper = VueTestInstance(BlankSlider, {
      options: {
        propsData: {
          config: DECLARATION_BLANK_ITEMS[3],
          data: DeclarationMock.declaration.personResponsibleForFinancialSettlement
        }
      }
    })
  })

  it('should match the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
