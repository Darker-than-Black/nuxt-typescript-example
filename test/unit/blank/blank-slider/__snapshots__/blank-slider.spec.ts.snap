// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blank-slider/index.vue should match the snapshot 1`] = `
<div class="slider">
  <div class="slider__item">
    <blank-subject-stub config="[object Object]" data="[object Object]">
      <slider-nav-stub slidersize="2" class="ml-2"></slider-nav-stub>
    </blank-subject-stub>
  </div>
</div>
`;
